website/content/docs/platform/k8s/helm/examples/ha-tls.mdx

```diff
@@ -0,0 +1,68 @@
+---
+layout: 'docs'
+page_title: 'HA Cluster with TLS'
+sidebar_current: 'docs-platform-k8s-examples-ha-tls'
+description: |-
+  Describes how to set up a Raft HA Vault cluster with TLS certificate
+---
+
+# Raft HA Server with TLS 
+
+Follow the steps from the example [HA Vault Cluster with Integrated Storage](/docs/platform/k8s/helm/examples/ha-with-raft) to build the cluster.
+
+Follow the examples and instructions in [Standalone Server with TLS](/docs/platform/k8s/helm/examples/standalone-tls) to create a TLS certificate.
```

Comment:
```suggestion

## Before you start

1. Follow the steps in [HA Vault Cluster with Integrated Storage](/docs/platform/k8s/helm/examples/ha-with-raft) to build your cluster.
1. Follow the steps in [Standalone Server with TLS](/docs/platform/k8s/helm/examples/standalone-tls) to create a TLS certificate.
```

Suggest making it clear that these are (essentially) prerequisites to following the instructions you provide below.